### ISSUE TITLE
Add extime.Time.asRFC1123()

### ISSUE THREAD

**Issue by ldanielburr** [twisted/epsilon#8]
When using extime.Time as part of a codebase that deals with setting HTTP cookies, it is useful to be able to format the Time object according to RFC1123, as that format is required when setting the cookie's "expires" value.

extime.Time.asRFC2822 produces similar output, with the exception of the timezone, which must be "GMT" in order to satisfy the requirements for a cookie's "expires" value.

**Comment by ldanielburr** [comment]
Quick sketch:
```python
def asRFC1123(self):
    from email.utils import formatdate
    return formatdate(self.asPOSIXTimestamp())
```

**Comment by ldanielburr** [comment]
Pull Request #11 created.

**Comment by mithrandi** [comment]
Merged in bce2120aa4b89f3cd33c700b339106ebd3eef03d, closing this manually because GitHub didn't do it.
